desktop/libs/notebook/src/notebook/management/commands/dbproxy_server.py

```diff
@@ -36,7 +37,10 @@ def handle(self, *args, **kwargs):
 
     cmd = os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", "..", "librdbms", "java", "bin", "dbproxy")
 
-    LOG.info("Executing %r (%r) (%r)" % (args, args, env))
+    command = ['dbproxy']
+    command.append(DBPROXY_EXTRA_CLASSPATH.get())
```

Comment:
Actually, why not just?
if DBPROXY_EXTRA_CLASSPATH.get(): 
  env['CLASSPATH'] = '%s:%s' (DBPROXY_EXTRA_CLASSPATH.get(), env.get('CLASSPATH', '')
?